assets/helpers/flashSale.js

```diff
@@ -419,6 +419,16 @@ function getFormattedFlashSalePrice(
   return fixDecimals(sale.saleDetails[countryGroupId].price);
 }
 
+function countdownTimerActive(flashSaleActive: boolean, showForHowManyDays: number, endTime: number): boolean {
+  if (flashSaleActive) {
+    const now = new Date();
```

Comment:
```suggestion
    const timeTravelDays = getTimeTravelDaysOverride();
    const now = timeTravelDays ? Date.now() + (timeTravelDays * 86400000) : Date.now();
```

this'll allow the time travel parameter to work